apps/system/js/haiku_emoji_panel.js

```diff
@@ -8,6 +8,18 @@
   };
   var mozMessage = navigator.mozMobileMessage;
   var logStatus = document.querySelector('#emoji-status-bottom');
+  var handleNextSMS = true;
+  //sets handleSMS to true and enables hover animations
+  function setNextTrue() {
+    handleNextSMS = true;
+    document.getElementById('emoji_left').style.pointerEvents = 'auto';
+    document.getElementById('emoji_middle').style.pointerEvents = 'auto';
+    document.getElementById('emoji_right').style.pointerEvents = 'auto';
```

Comment:
we can add a class in css for pointer-events: none; and can remove that class from target elements here